src/scales/scale.time.js

```diff
@@ -383,7 +383,7 @@ function computeOffsets(table, ticks, min, max, options) {
 		}
 	}
 
-	return {left: left, right: right};
+	return options.ticks.reverse ? {left: right, right: left} : {left: left, right: right};
```

Comment:
Not directly related to this PR but maybe the good timing to change it: since we use these values for vertical and now reversed scales, it may be easier to understand if we rename those properties for `start` and `end`:

```javascript
return options.ticks.reverse ? {start: start, end: end} : {start: end, end: start};
```